lib/src/widget/nav_bar/data/nav_bar_config.dart

```diff
@@ -19,40 +19,59 @@ class NavBarConfig extends Equatable {
   final bool isWidthExpanded;
   final NavBarType type;
 
+  /// If [true] than [NavBar] will be respect the keyboard height
+  /// (it will be added to the default bottom padding)
+  final bool showAboveKeyboard;
```

Comment:
💅 Here we can maybe say something like:
`If [true] then the height of the keyboard will be added to the bottom padding of the [NavBar], so that it will be shown above the keyboard` 
:)

Reply:
Sure, updated 😉